circle.yml

```diff
@@ -27,8 +27,7 @@ test:
           - server/**/*.jsx
     - MOCHA_FILE=./test-results-client.xml npm run test-client -- -R mocha-junit-reporter -t $CIRCLE_NODE_TOTAL -i $CIRCLE_NODE_INDEX:
         parallel: true
-    - MOCHA_FILE=./test-results-server.xml npm run test-server -- -R mocha-junit-reporter -t $CIRCLE_NODE_TOTAL -i $CIRCLE_NODE_INDEX:
-        parallel: true
+    - npm run test-server:ci
```

Comment:
With this line tests are executed only on node 0, which isn't what I would expect. This needs to be further investigated.